package.json

```diff
@@ -72,10 +72,10 @@
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
     "@emotion/core": "^10.0.7",
-    "@storybook/addon-options": "~5.1.11",
-    "@storybook/addon-storysource": "~5.1.11",
-    "@storybook/addons": "~5.1.11",
-    "@storybook/react": "~5.1.11",
+    "@storybook/addon-options": "^5.3.18",
+    "@storybook/addon-storysource": "^5.3.18",
+    "@storybook/addons": "^5.3.18",
```

Comment:
Can you check if we really need all those packages of addons? I think we need the story source one, not sure why we will need the others, see if we can do some package optimizations here.
Also, I think I read somewhere that story source might be deprecated and it is recommended to use another package, but I can't find it anywhere so I hope I'm not imagining anything.

Reply:
Sure!!